intro-Linux/ch_uploading_files.tex

```diff
@@ -2,25 +2,34 @@ \chapter{Uploading/downloading/editing files}
 
 \subsection{Uploading/downloading files}
 
-Detailed information about uploading/downloading files with ``scp'' (using an OS
-X or Linux client) or WinSCP (Windows client), see
-\url{http://hpc.ugent.be/userwiki/index.php/User:VscCopy}.
+To transfer files from and to the HPC, see
+\href{\HPCManualURL#sec:filetranfer}{the section about tranferring files in chapter 3 of the HPC manual}.
+
+\ifwindows
 
 After copying files it is advised to run
 
 \begin{prompt}
 %\shellcmd{dos2unix filename}
 \end{prompt}
 
-as this will fix any problems with windows / unix conversion.
+as this will fix any problems with Windows/Unix conversion. If you don't do this,
+you might get errors like this one:
 
+\begin{prompt}
+%\shellcmd{qsub fibo.pbs}%
+qsub:  script is written in DOS/Windows text format
```

Comment:
This is also one to mention in the Troubleshooting section, i.e. just mention this error message, and then refer to here (so you'll need to make a Windows-specific  `subsubsection`).

It makes to do that in this PR imho (even though it means touching another chapter), but let's merge #227 first.

Reply:
Done